frontend/src/components/tutorial/Tutorial.js

```diff
@@ -33,7 +33,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function Tutorial({ fixedPosition, pointerRefs, nextStepTriggeredBy, hubName }) {
+export default function Tutorial({
+  fixedPosition,
+  pointerRefs,
+  nextStepTriggeredBy,
+  hubName,
+  handleTabChange,
+  typesByTabValue,
```

Comment:
Nit: WDYT about sorting imports and props (when possible)? I find it helps with readability and thereby maintainability.
```suggestion
  fixedPosition,
  hubName,
  handleTabChange,
  nextStepTriggeredBy,
  pointerRefs,
  typesByTabValue,
```

VS Code makes it really easy to do with `Cmd+shift+p` and then typing "sort" 🎉